### PR TITLE
doc: Provide changelog for version 1.0.0

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Review by zimlu02 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,46 @@
...
+
+### Features
+* Add an ability to enable/disable analysis features ([d55dd09](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/d55dd09c8df188437bb836d19223cc7488f8f579))
+* Add line commenting ability ([f4083a3](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f4083a3576ca066545929993b51593dd2d40d08c))
+* Add IDMS snippets [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([af98cbc](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/af98cbcacccd9b7a89a3d4da1da3c64b4af62776))
+* Adjust level numbers for COPY IDMS ([3757275](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/375727583dd559ecf216bed5d3e065953b8a354f))
+* Allow paths for different types of copybooks ([47a88f7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/47a88f72249b7721e43759f810a25fff55f251a6))
+* Enhance the support for obsolete code underscoring ([6749d3d](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/6749d3d9c601b8baa16f8d00d79c3ddd2b2f41eb))
```
Comment (grianbrcom) [inline CHANGELOG.md:33]: I think that it was a refactoring.

**Review by grianbrcom — APPROVED** [review]
🌪️

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,46 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.22.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.21.0...0.22.0) (2022-03-21)
```
[the remark this answers is not in the document]
Comment (grianbrcom) [reply CHANGELOG.md:4]: Agree. We need it here.

[the remark this answers is not in the document]
Comment (zimlu02) [reply CHANGELOG.md:4]: good catch Zeb! 
Also the date will change...

**Review by zimlu02 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (temanbrcom) [reply CHANGELOG.md:33]: Nice catch

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (temanbrcom) [reply CHANGELOG.md]: Thanks, I updated the version. The date will be updated automatically.

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**